config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -126,7 +126,7 @@ periodics:
       - --deployment=node
       - --gcp-project-type=node-e2e-project
       - --gcp-zone=us-west1-b
-      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/perf-image-config.yaml
+      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config.yaml
```

Comment:
btw, this is why perf tests were executing: https://github.com/kubernetes/test-infra/blob/caeaacec0de5a867e5d420da85c8f69bf33fc1ca/jobs/e2e_node/perf-image-config.yaml#L8-L9

Reply:
Oh got it! thanks a lot.

Reply:
this is very illogical.

Reply:
So, the internal config pointed to `Node Performance Testing`. Something that I should have gone down a level deeper.